.github/workflows/e2e.yml

```diff
@@ -56,6 +59,11 @@ jobs:
         uses: ./.github/actions/sanity-check
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Set up JDK ${{ matrix.java }}
+        uses: actions/setup-java@v2
+        with:
+          java-version: ${{ matrix.java }}
+          distribution: 'adopt'
```

Comment:
This only setup jdk of the host. You should also modify the runtime environment of e2e containers at the same time.

Reply:
Thanks @SbloodyS !
I've updated the PR.